golang/cosmos/app/app.go

```diff
@@ -788,25 +788,19 @@ func NewAgoricApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 	app.SetEndBlocker(app.EndBlocker)
 
-	const (
-		upgradeName     = "agoric-upgrade-14"
-		upgradeNameTest = "agorictest-upgrade-14"
-	)
+	for name := range upgradeNamesOfThisVersion {
```

Comment:
It'd be more in keeping with the Agoric style to define `upgradeNamesOfThisVersion` before using it.

Reply:
I'd agree, but it would no longer be collocated with the upgrade handler, nor with their registration